framework/src/loops/ComputeThreadedGeneralUserObjectsThread.C

```diff
@@ -48,3 +49,17 @@ ComputeThreadedGeneralUserObjectsThread::operator()(const GeneralUserObjectRange
     caughtMooseException(e);
   }
 }
+
+void
+ComputeThreadedGeneralUserObjectsThread::printGeneralExecutionInformation(
+    const GeneralUserObjectRange & range) const
+{
+  // TODO: Threaded UOs dont know their thread number so this will print too often
```

Comment:
This is pretty dumb that don't have a `_tid` member in this class.

Reply:
yeah right